### DISCUSSION TITLE
Replying to a message without event

### DISCUSSION THREAD

**Discussion by BuzzMoody** [discord-php/DiscordPHP#1188] [Q&A] [answered]
Basically have the [messageid] stored and I want to be able to reply back it in the future but having now luck with the following:

```php
$guild = $discord->guilds->get('id', [guildid]);
$channel = $guild->channels->get('id', [channelid]);
$channel->messsages->fetch([messageid], true)->done(function (Message $message) {
   $message->reply("Reply to [messageid]");
});
```

$guild and $channel are returning values, but it never returns the promise. Would love some insight into how this should be fone.

**Comment by SQKo** [comment, marked as answer]
> Basically have the [messageid] stored and I want to be able to reply back it in the future but having now luck with the following:
> 
> ```
> $guild = $discord->guilds->get('id', [guildid]);
> $channel = $guild->channels->get('id', [channelid]);
> $channel->messsages->fetch([messageid], true)->done(function (Message $message) {
>    $message->reply("Reply to [messageid]");
> });
> ```
> 
> $guild and $channel are returning values, but it never returns the promise. Would love some insight into how this should be fone.

You have typo with triple s in `messages`


```php
$guild = $discord->guilds->get('id', [guildid]);
$channel = $guild->channels->get('id', [channelid]);
$channel->messages->fetch([messageid], true)->then(function (Message $message) {
   return $message->reply("Reply to [messageid]");
})->done();
```
